python-threatexchange/threatexchange/signal_type/signal_base.py

```diff
@@ -11,7 +11,6 @@
 import typing as t
 
 from .. import common
-from .. import dataset
```

Comment:
When the commit message is 25 times longer than the fix itself, mistakes have been made... haha